pyproject.toml

```diff
@@ -1,3 +1,9 @@
+[build-system]
```

Comment:
I'm not super familiar with pyproject.toml. Does this automatically get installed when you use pip or do you need another tool?

Reply:
gets installed: https://pip.pypa.io/en/stable/reference/pip/#pep-517-and-518-support

Reply:
good to know, thanks!

Reply:
yeah, I'm learning as I go here too.  so don't necessarily assume I know what I'm doing! 😅 would be great if you get an opportunity to pull the PR, and try to break it locally.

Reply:
But note also #1246...

Reply:
this PR includes [support for legacy behavior](https://github.com/napari/napari/pull/1287/files#diff-2eeaed663bd0d25b7e608891384b7298R95-R96) and doesn't require pyproject.toml